basic/scraper.go

```diff
@@ -25,17 +27,16 @@ type Scraper struct {
 	ch        chan<- prometheus.Metric
 
 	// internal
-	svc         *cloudwatch.CloudWatch
+	svc         *cloudwatchV2.Client
```

Comment:
```suggestion
	svc         *cloudwatch.Client
```
IMO, there is no need to make the version part of the package name unless there is a name collision. The same would apply to "aws" and "types" packages.